rpc_shutdown/DOCS.md

```diff
@@ -64,7 +64,7 @@ Use the following inside Home Assistant service call to use it:
 
 ```yaml
 service: hassio.addon_stdin
-data:
+service_data:
```

Comment:
This change is incorrect, service calls in Home Assistant have a data object.

Considering your PR description, it sounds like you are trying to add something else, but that is not related to this part of the documentation.